cypress/fixtures/stubbing/ScenarioSharing/workspaces.js

```diff
@@ -18,9 +19,25 @@ export const WORKSPACE_NO_USERS = {
   id: 'W-stbbdbwry0',
   key: 'DemoBreweryNoUsers',
   security: {
-    default: '',
+    default: ROLES.SCENARIO.VIEWER,
```

Comment:
Not sure what is intended with the `WORKSPACE_NO_USERS` workspace, but setting its default role to 'viewer' will give "read" permission to all users (whereas the variable name seems to convey that we don't want users to have access to this workspace)

Reply:
I will set it to 'none'